SITCOM-1397_Extract_dominant_frequencies_from_sound_waves_in_video_associated_with_vibrations_on_Top-End_Assembly_II.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<ul><li>Please consider using scipy instead of librosa</li><li>Perhaps try with a larger window n_fft = 4096 for instance. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I have tried to use different window frames for this but it doesn't change the results. It is open for users to change it.